packages/client/src/index.ts

```diff
@@ -1,48 +1,9 @@
-import { assertGlobalPresent, assertProvider, assertProviderManifest } from './assert.js';
-import { isPenumbraManifest, type PenumbraManifest } from './manifest.js';
-import type { PenumbraProvider } from './provider.js';
-import { PenumbraSymbol } from './symbol.js';
-
-declare global {
-  interface Window {
-    /** Records injected upon this global should be identified by a name matching
-     * the origin segment of their manifest href `PenumbraProvider['manifest']`. */
-    readonly [PenumbraSymbol]?: undefined | Readonly<Record<string, PenumbraProvider>>;
-  }
-}
-
-/** Return the specified provider, without verifying anything. */
-export const getPenumbraUnsafe = (penumbraOrigin: string): PenumbraProvider | undefined =>
-  window[PenumbraSymbol]?.[penumbraOrigin];
-
-/** Return the specified provider after confirming presence of its manifest. */
-export const getPenumbra = (penumbraOrigin: string): Promise<PenumbraProvider> =>
-  assertProvider(penumbraOrigin);
-
-/** Fetch the specified provider's manifest. */
-export const getPenumbraManifest = async (
-  penumbraOrigin: string,
-  signal?: AbortSignal,
-): Promise<PenumbraManifest> => {
-  const manifestJson = await assertProviderManifest(penumbraOrigin, signal);
-  if (!isPenumbraManifest(manifestJson)) {
-    throw new TypeError('Invalid manifest');
-  }
-  return manifestJson;
-};
+export * from './error.js';
+export * from './event.js';
+export * from './event-listener.js';
 
-/** Fetch all manifests for all providers available on the page. */
-export const getAllPenumbraManifests = (
-  signal?: AbortSignal,
-): Record<string, Promise<PenumbraManifest>> =>
-  Object.fromEntries(
-    Object.keys(assertGlobalPresent()).map(providerOrigin => [
-      providerOrigin,
-      getPenumbraManifest(providerOrigin, signal),
-    ]),
-  );
+export { PenumbraClient } from './client.js';
```

Comment:
suggestion: export `createPenumbraClient` too and change `static` methods to `public`. The `createPenumbraClient` makes the API consistent and finished. 

Consider two usages:

```ts
// penumbra.ts
import { PenumbraClient } from '@penumbra-zone/client';
export const client = PenumbraClient.create();

// hooks.ts
import { PenumbraClient } from '@penumbra-zone/client';
import { client } from './penumbra';

const reconnect = async () => {
    const providers = PenumbraClient.getAllProviderOrigins();
    const connected = providers.find((origin) => PenumbraClient.isConnected(origin));
    if (!connected) return;
    await client.connect(connected);
};
```

and

```ts
// penumbra.ts
import { createPenumbraClient } from '@penumbra-zone/client';
export const client = createPenumbraClient();

// hooks.ts
import { client } from './penumbra';

const reconnect = async () => {
    const providers = client.getAllProviderOrigins();
    const connected = providers.find((origin) => client.isConnected(origin));
    if (!connected) return;
    await client.connect(connected);
};
```

the first example is taken straight from https://github.com/penumbra-zone/nextjs-penumbra-client-example/pull/6. The difference seems minor, and still it complicates the DX a little. Why using `PenumbraClient`'s static methods when `client` is already created? And it will always be created.

and a nitpick: maybe it's just me but when I see libraries that make me use classes, I want to find another library instead. Classes always look old-fashioned, idk.

Reply:
ok, I'd say that making static methods also public could be a non-breaking addition for the future. Let's way for what our users have to say and, if they would have this feedback, we'll implement it.

can't see the `createPenumbraClient` for now

Reply:
removed the static effective methods `PenumbraClient.create` and `PenumbraClient.connect` in favor of your suggested `createPenumbraClient`

this feels much cleaner. now all static methods are 'read-only', and only instance methods will take action.